brickflow/engine/workflow.py

```diff
@@ -117,8 +117,8 @@ class Workflow:
     schedule_pause_status: str = "UNPAUSED"
     default_cluster: Optional[Cluster] = None
     clusters: List[Cluster] = field(default_factory=lambda: [])
-
     health: Optional[List[JobsHealthRules]] = None
+    timeout_seconds: Optional[float] = None
```

Comment:
shouldnt this be int? and not a floating point number?

Reply:
@stikkireddy I defined it as a float based on your model: https://github.com/Nike-Inc/brickflow/blob/main/brickflow/bundles/model.py#L1664 would you like me to update the model as well?

Reply:
After a conversation with @asingamaneni I've changed this in the model from 'float' to 'int.' 
This did not effect unit tests passing even when values were left as floats.  

I want to test with a real actual deploy that has the value set as a float before this is merged.

Reply:
So I worked with Srinivas Caveri, we changed the Workflow and TaskSettings definition in the bundle model and in the Workflow class from 'float' to 'int'.

We then performed a deploy with a task timeout set to "2400.5" and the deploy went through and the task was set to 40 minutes and 0 seconds which was the expected outcome. 

That confirms that if any existing customer somehow had a timeout value set to a decimal number of seconds this change will be none disruptive to them.